dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -132,7 +140,20 @@ public String toString() {
   public int getErrorCount() {
     return getCauseToSourceClasses().size();
   }
+  
+  public int getTargetClassCount() {
```

Comment:
I'd recommend Javadoc for all public methods.

Reply:
done